src/main/java/ch/uzh/ifi/hase/soprafs22/entity/Chat.java

```diff
@@ -46,17 +45,30 @@ public List<Message> getMessages(int fromIndex, int toIndex) {
     // now, fromIndex must be < size, and toIndex too.
     return messages.subList(fromIndex, toIndex);
   }
+    /**
+     * Gets one Message from the chat
+     * @param index: the index of the message
+     * @return Message: the Message on position index
+     */
+    public Message getMessages(int index){
+        if (messages.isEmpty()){
+            return null;
+        }
+        return messages.get(index);
```

Comment:
Also, make sure not to raise indexErrors here (see getMessages from above) (Recommendation)